src/EA.Weee.Web/Areas/Scheme/Controllers/ManageEvidenceNotesController.cs

```diff
@@ -280,6 +290,28 @@ public async Task<ActionResult> ViewEvidenceNote(Guid pcsId, Guid evidenceNoteId
             }
         }
 
+        [HttpGet]
+        public async Task<ActionResult> DownloadEvidenceNote(Guid evidenceNoteId)
+        {
+            using (var client = apiClient())
+            {
+                var request = new GetEvidenceNoteForAatfRequest(evidenceNoteId);
+
+                var result = await client.SendAsync(User.GetAccessToken(), request);
+
+                var model = mapper.Map<ViewEvidenceNoteViewModel>(new ViewEvidenceNoteMapTransfer(result, null, true));
```

Comment:
@GuidoWorkDev You can pass a property to the model builder to indicate its a PDF by doing

var model = mapper.Map<ViewEvidenceNoteViewModel>(new ViewEvidenceNoteMapTransfer(result, null, true){
IsPrinable = true
);

This can be used in the view to output the histotry in the correct place

PrintableVersion